api/envoy/config/ratelimit/v2/rls.proto

```diff
@@ -11,22 +11,12 @@ import "validate/validate.proto";
 
 // Rate limit :ref:`configuration overview <config_rate_limit_service>`.
 message RateLimitServiceConfig {
-  oneof service_specifier {
```

Comment:
I think we still need to keep the `oneof` and have both the `reserved` and the `grpc_service` fields under it to avoid potential wire issues. I'd confirm with @mattklein123.

Reply:
If I understand this https://developers.google.com/protocol-buffers/docs/proto3#backwards-compatibility-issues correctly  I think it is OK to remove oneof for required fields. But will wait for @mattklein123 confirmation.

Reply:
I think it's OK to remove oneof in this case. @htuch ?

Reply:
Yep, this one is safe.